osxkeychain/osxkeychain_darwin.go

```diff
@@ -12,10 +12,10 @@ import (
 	"errors"
 	"net/url"
 	"strconv"
-	"strings"
 	"unsafe"
 
 	"github.com/docker/docker-credential-helpers/credentials"
+	"strings"
```

Comment:
Ah, I guess it's because docker/docker follows the `goimport` conventions, in which case they are [grouped, and sorted](https://groups.google.com/forum/#!topic/golang-nuts/Ir0-hKNkEkQ), but `gofmt` won't complain if they were not grouped.

> gofmt follows a simple rule, which is to sort each group of imports, 
> where different groups are separated by a blank line. 
>
> goimports also follows a simple rule, which is to put stdlib imports 
> in one group and non-stdlib imports in a different group.